src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -230,63 +218,57 @@ public async Task GetInfoPackage()
             }
         }
 
-        private static void DumpStackTraces(ZipArchive archive, string prefix)
+        private static async Task WriteLogFile(ZipArchive archive)
         {
-            var zipArchiveEntry = archive.CreateEntry($"{prefix}/stacktraces.json", CompressionLevel.Optimal);
+            var prefix = $"{_serverWidePrefix}/{DateTime.UtcNow:yyyy-MM-dd H:mm:ss}.txt";
 
-            var threadsUsage = new ThreadsUsage();
-            var sp = Stopwatch.StartNew();
-
-            using (var stackTraceStream = zipArchiveEntry.Open())
-            using (var sw = new StringWriter())
+            try
             {
-                try
-                {
-                    if (Debugger.IsAttached)
-                        throw new InvalidOperationException("Cannot get stack traces when debugger is attached");
+                var ms = new MemoryStream();
+                var isInfoEnabled = LoggingSource.Instance.IsInfoEnabled;
+                var isOperationsEnabled = LoggingSource.Instance.IsOperationsEnabled;
 
-                    ThreadsHandler.OutputResultToStream(sw);
+                if (isInfoEnabled == false)
```

Comment:
why not `LoggingSource.Instance.IsInfoEnabled = true` ?

Reply:
done